src/main/java/gift/Member.java

```diff
@@ -11,19 +11,21 @@ public class Member {
     @NotBlank(message = "비밀번호는 필수 입력값입니다.")
     private String password;
 
-    public void setId(Long id) {
+    public Member( Long id, String email, String password ) {
```

Comment:
```suggestion
    public Member(Long id, String email, String password) {
```

제가 불필요한 공백 찾는데 강점이 있어요 😂